src/types.rs

```diff
@@ -112,17 +118,21 @@ impl Configuration {
         let delay = Duration::from_millis(1000);
         let url = "https://resolver.holohost.net/update/addHost";
 
+        let host_id = crate::utils::get_host_id()?;
+
         let happ_urls = self
             .dnas
             .iter()
             .cloned()
             .filter(|dna| dna.holo_hosted)
             .filter_map(|dna| dna.happ_url)
```

Comment:
Is this the right information? We are expecting the happ ID (hash) not a url / domain name.

Reply:
u r right